### PR TITLE
Welcome Screen

### PR DESCRIPTION
**Pull request by sivaShilpa**
### Done
- Edited the App.tsx to add the homestack, test screen (keeping a test screen to test out new components)
- Added leftBackArrow icon pick and the welcome screen pic to assets
- Welcome Screen complete
- Buttons navigates to Login Screen and SignUp Screen
- Added test files for each new file. 

### To Do 
- Write functional test files instead of snap shots for each test files

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,24 +3,43 @@ import { StatusBar } from 'expo-status-bar';
...
-import ButtonComponent from './components/ButtonComponent';
+import WelcomeScreen from './screens/WelcomeScreen';
+import LoginScreen from './screens/LoginScreen';
+import SignUpScreen from './screens/SignUpScreen';
 
 const Tab = createBottomTabNavigator();
+const Stack = createNativeStackNavigator();
 
-const HomeScreen = () => {
+const TestScreen = () => {
```
Comment (lindentree) [inline src/App.tsx:18]: Why did you change this name to TestScreen?

Comment (sivaShilpa) [reply src/App.tsx:18]: Because now we have the Welcome Screen as the Home Screen. Last time I deleted the Home Screen. But this time I thought I will keep it as a Test Screen in case if we want to use it to test out any reusable component we create.

Comment (sivaShilpa) [reply src/App.tsx:18]: If you want I can delete it.

Comment (MelisaIm) [reply src/App.tsx:18]: @sivaShilpa I don't think we need it :) if folks want to test something out, they can add it in their branch. Alternatively, it could be behind some env configuration so it's not visible all the time but we can create a separate task for that 👍🏽

**Review by MelisaIm — APPROVED** [review]
lgtm! Just remove the test route and component for now, and we can define a separate task for something like it :)

**Comment by sivaShilpa** [comment]
Made changes to remove the test screen and route. Merging and deleting branch.
